From f2e28facd1940940b42597ea2654d21b0a09dd1a Mon Sep 17 00:00:00 2001
From: "Lori A. Burns" <lori.burns@gmail.com>
Date: Thu, 3 Apr 2025 17:40:15 -0400
Subject: [PATCH] avoid headers in windows

---
 src/cavity/GePolCavity.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/cavity/GePolCavity.cpp b/src/cavity/GePolCavity.cpp
index 857cc40b..3f19396e 100644
--- a/src/cavity/GePolCavity.cpp
+++ b/src/cavity/GePolCavity.cpp
@@ -33,7 +33,9 @@
 #include <vector>
 
 #include <sys/types.h>
+#ifndef _MSC_VER
 #include <unistd.h>
+#endif
 
 #include "Config.hpp"
 
-- 
2.40.1

